src/Sylius/Component/Order/Model/Order.php

```diff
@@ -344,8 +326,6 @@ public function addAdjustment(AdjustmentInterface $adjustment)
             $adjustment->setAdjustable($this);
```

Comment:
This should be removed as item __must__ be assigned to some "parent", otherwise it's broken.